lib/ace/ext/papeeria/tex_completer.coffee

```diff
@@ -162,8 +170,9 @@ define((require, exports, module) ->
           @refGetter.getReferences(@referencesUrl, callback)
         else switch context
           when "start" then callback(null, BASIC_SNIPPETS.concat(LIST_SNIPPET,
-            EQUATION_ENV_SNIPPETS, REFERENCE_SNIPPET))
-          when LIST_STATE then callback(null, LIST_KEYWORDS.concat(EQUATION_ENV_SNIPPETS, LIST_SNIPPET, LIST_END_ENVIRONMENT))
+            EQUATION_ENV_SNIPPETS, REFERENCE_SNIPPET, CITATION_SNIPPET))
+          when LIST_STATE then callback(null, LIST_KEYWORDS.concat(EQUATION_ENV_SNIPPETS,
+            LIST_SNIPPET, LIST_END_ENVIRONMENT, CITATION_SNIPPET, REFERENCE_SNIPPET))
```

Comment:
@Mikhail-M to be clear, please do reorder concat arguments so that the difference between them was obvious to human. The same order of cmmon arguments in both concat calls.